homeassistant/components/google_assistant/trait.py

```diff
@@ -326,15 +326,17 @@ def sync_attributes(self):
             response["colorModel"] = "hsv"
 
         if features & light.SUPPORT_COLOR_TEMP:
+
+            def convert(mireds):
+                if mireds is None:
+                    return None
+                return color_util.color_temperature_mired_to_kelvin(mireds)
+
             # Max Kelvin is Min Mireds K = 1000000 / mireds
             # Min Kelvin is Max Mireds K = 1000000 / mireds
             response["colorTemperatureRange"] = {
-                "temperatureMaxK": color_util.color_temperature_mired_to_kelvin(
-                    attrs.get(light.ATTR_MIN_MIREDS)
-                ),
-                "temperatureMinK": color_util.color_temperature_mired_to_kelvin(
-                    attrs.get(light.ATTR_MAX_MIREDS)
-                ),
+                "temperatureMaxK": convert(attrs.get(light.ATTR_MIN_MIREDS)),
```

Comment:
Please revert this change now. It's no longer needed now that https://github.com/home-assistant/architecture/issues/297 has been implemented.